### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by devendra-alt**
### 👨‍💻  Requirements covered
☑️ Used database transitions.
☑️ Modified and deleted data.
☑️ Used complex queries to harvest analytical questions.
☑️ Added `species` COLUMN on the `animals` table.
☑️ Populated `animals` table with given data.
☑️ Updated `ID` with `SERIAL` and `PRIMARY KEY` type.

psq_transition1
psq_transition2
psq_transition3
psq_transition4
psq_transition5
psq_transition6
psq_transition7
psq_transition8
psq_transition9

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,28 @@ SELECT name FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```
Comment (V-Blaze) [inline queries.sql:38]: - You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: 
>Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Verify that changes were made.
Commit the transaction.
Verify that changes persist after commit.

- [x] You did not have the queries for the above requirements in your `queries.sql` file. I'll suggest you add the queries to the file. :smile: :small_airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @devendra-alt 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Queries are written correctly :heavy_check_mark: 
- Readme looks professional :heavy_check_mark: 
- Correct Git flow :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (devendra-alt) [reply queries.sql:38]: thank you @V-Blaze 💯 for your valuable feedback

**Review by mukhammadrizooff — APPROVED** [review]
Hi @devendra-alt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**